packages/mdc-top-app-bar/short/foundation.ts

```diff
@@ -45,33 +47,65 @@ export class MDCShortTopAppBarFoundation extends MDCTopAppBarBaseFoundation {
       this.adapter_.addClass(cssClasses.SHORT_HAS_ACTION_ITEM_CLASS);
     }
 
-    // this is intended as the short variant must calculate if the
-    // page starts off from the top of the page.
-    this.handleTargetScroll();
+    // If initialized with SHORT_COLLAPSED_CLASS, the bar should always be collapsed
+    this.setAlwaysCollapsed(
+      this.adapter_.hasClass(cssClasses.SHORT_COLLAPSED_CLASS));
+  }
+
+  /**
+   * Set if the short top app bar should always be collapsed.
+   *
+   * When set to `true`, the bar collapses.
+   * When set to `false`, the bar will determine if it should collapse based on scroll position.
+   */
+  setAlwaysCollapsed(value: boolean) {
+    this.isAlwaysCollapsed_ = !!value;
+    if (this.isAlwaysCollapsed_) {
+      this.collapse_();
+    } else {
+      // let maybeCollapseBar_ determine if the bar should be collapsed
+      this.maybeCollapseBar_();
+    }
+  }
+
+  getAlwaysCollapsed() {
+    return this.isAlwaysCollapsed_;
   }
 
   /**
    * Scroll handler for applying/removing the collapsed modifier class on the short top app bar.
    * @override
    */
   handleTargetScroll() {
-    if (this.adapter_.hasClass(cssClasses.SHORT_COLLAPSED_CLASS)) {
+    this.maybeCollapseBar_();
+  }
+
+  private maybeCollapseBar_() {
+    if (this.isAlwaysCollapsed_) {
```

Comment:
This condition is already covered in `setAlwaysCollapsed`? is this specific to `handleTargetScroll`?

Reply:
Yes. This will short circuit the scroll handler, and keep the bar collapsed.